inference-engine/src/mkldnn_plugin/nodes/mkldnn_mvn_node.cpp

```diff
@@ -1448,4 +1494,29 @@ bool MKLDNNMVNNode::created() const {
     return getType() == MVN;
 }
 
+bool MKLDNNMVNNode::checkAxesSuitability(const std::shared_ptr<const ngraph::Node>& node) {
+    const auto mvn = std::dynamic_pointer_cast<const ngraph::op::v6::MVN>(node);
+    if (mvn != nullptr && node->get_input_size() == 2) {
+        if (auto axesNode = dynamic_cast<ngraph::op::v0::Constant*>(mvn->get_input_node_ptr(1))) {
+            auto axesVal = axesNode->cast_vector<int>();
+            auto& mvnShape = mvn->get_output_shape(0);
+            if (mvnShape.size() == 1) {
+                if (axesVal.size() == 1 && axesVal[0] == 0)
+                    return true;
+                else
+                    return false;
+            }
+            if (mvnShape.size() > 5 || (mvnShape.size() != axesVal.size() + 1 && mvnShape.size() != axesVal.size() + 2))
+                return false;
+            int value = mvnShape.size() - 1;
+            for (int i = axesVal.size() - 1; i >= 0; i--, value--) {
```

Comment:
Not sure but seems like it will disable decomposition for cases which are not supported by CPU MVN impl like [0, 1, 3]. Please correct me if I am wrong.

Reply:
Sorry. Seems like I was wrong. It will enable decomposition for "valid" cases as [0, 2, 3], won't it?

Reply:
0 - is not valid axis for CPU implementation, due to all calculation executed per batch.

Reply:
Ok, I see. To be honest from this function really hard to understand which cases are supported and which not. Can we at least add a comment with the list of supportes cases like:
- 1D: axes: [0]
- 2D: axes: [1]
- 3D: axes: [1,2], [2]
- 4D: axes: [1,2,3], [2,3]
- 5D: axes: [1,2,3,4], [2,3,4]

Reply:
Added